root/layout/components/sidebar/SidebarType.js

```diff
@@ -24,7 +24,10 @@ type Props = {|
 const SidebarType = ({$c, entity, typeType}: Props) => (
   entity.typeID ? (
     <SidebarProperty className="type" label={l('Type:')}>
-      {l_attributes($c.linked_entities[typeType][entity.typeID].name)}
+      {lp_attributes(
+        $c.linked_entities[typeType][entity.typeID].name,
+        typeType,
+      )}
```

Comment:
Kept it as similar as possible to the original, but wouldn't it make sense to just change from typeID to typeName and use entity.typeName instead of looking into CatContext?

Reply:
I believe it saves a bit of memory to avoid duplicating the types on pages with tons of entities, though I haven't measured it to be honest. The main issue is that `$c.linked_entities[typeType][entity.typeID].name` is ugly and verbose. There are ways I think we can hide that, like defining a `localizeTypeName($c, entity)` utility.